packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -935,135 +714,100 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
    * @returns AccountAuthenticator
    */
   async signTransaction(
-    transactionOrPayload: AnyRawTransaction | Types.TransactionPayload,
+    transactionOrPayload: AnyRawTransaction | InputTransactionData,
     asFeePayer?: boolean,
-    options?: InputGenerateTransactionOptions,
+    options?: InputGenerateTransactionOptions & {
+      expirationSecondsFromNow?: number;
+      expirationTimestamp?: number;
+    }
   ): Promise<AccountAuthenticator> {
```

Comment:
`AptosSignTransactionOutputV1_1` returns the raw transaction on addition to the AccountAuthenticator, but we fails to simply send the raw transaction back to the dapp, need to deserialize probably